contracts/Libraries/NTransferUtilV1.sol

```
@@ -24,6 +28,10 @@ library NTransferUtilV1 {
     address recipient,
     uint256 amount
   ) public {
+    if (amount == 0) {
+      return;
```

Instead of silently returning, this function show throw an error.